modules/rules_java/6.5.1/MODULE.bazel

```diff
@@ -2,6 +2,8 @@ module(
     name = "rules_java",
     version = "6.5.1",
     compatibility_level = 1,
+    # Requires @bazel_tools//tools/jdk:bootstrap_runtime_toolchain_type.
+    bazel_compatibility = [">=6.4.0"],
```

Comment:
Wow, that's not even released yet, seems like maybe it wasn't a wise decision in rules_java. Should we change it there instead?

Reply:
I guess the latest rolling releases should still work with rules_java 6.5.1?

Reply:
Yes, they do work, actually quite a few of them: https://github.com/bazelbuild/bazel/commit/8715e9ae6f5c2fd42607018b3adaad14bf601a1c